hamilton/base.py

```diff
@@ -371,6 +404,13 @@ def build_result(**outputs: Dict[str, Any]) -> np.matrix:
         # Create the matrix with columns as rows and then transpose
         return np.asmatrix(list_of_columns).T
 
+    def inputs_types(self) -> List[Type[Type]]:
```

Comment:
why no optional? the others above have it.

Reply:
Cause it won't return `None`, and its a valid implementation of the superclass method